user-onboarding/cypress/e2e/form_test.cy.js

```diff
@@ -0,0 +1,273 @@
+const firstNameInput = () => cy.get('input[name="first_name"]')
+const lastNameInput = () => cy.get('input[name="last_name"]')
+const emailInput = () => cy.get('input[name="email"]')
+const passInput = () => cy.get('input[name="password"]')
+const confirmPassInput = () => cy.get('input[name="confirmPassword"]')
+const termsCheckbox = () => cy.get('input[name="terms"]')
+//const submitButton = () => cy.get('button[class="submit"]') - same as below:
+const submitButton = () => cy.get('.submit')
+const firstNameValErr = () => cy.get('div[class="error-first"]')
+const lastNameValErr = () => cy.get('div[class="error-last"]')
+const emailValErr = () => cy.get('div[class="error-email"]')
+const pwValErr = () => cy.get('div[class="error-pw"]')
+const confirmPwValErr = () => cy.get('div[class="error-confirmPw"]')
+
+describe('User Onboarding MVPs', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:3000')
+  })
+
+  it('name input', () => {
+    firstNameInput()
+      .should("have.value", "")
+      .type('Marele')
+      .should('have.value', 'Marele')
+
+    lastNameInput()
+      .should('have.value', '')
+      .type('Stormwind')
+      .should('have.value', 'Stormwind')
+  });
+
+  it('email input', () => {
+    emailInput()
+      .should('have.value', '')
+      .type('marele@stormwind.com')
+      .should('have.value', 'marele@stormwind.com')
+  });
+
+  it('password/confirm input', () => {
+    passInput()
+      .should('have.value', '')
+      .type('M@releSt0rmw1nd')
+      .should('have.value', 'M@releSt0rmw1nd')
+
+    confirmPassInput()
+      .should('have.value', '')
+      .type('M@releSt0rmw1nd')
+      .should('have.value', 'M@releSt0rmw1nd')
+  });
+
+  it('terms of service checkbox can be clicked', () => {
+    termsCheckbox()
+      .should('not.be.disabled')
+      .click()
+    //Error occurs - expects the boolean to be true, but it is false; even though this is not part of the assignment, I would like to go back to this in my code and see why this is happening as the state is changing from false to true when I inspect it.
```

Comment:
Maybe I'm not understanding what you're trying to test here. The checkbox will not be disabled no matter if the box is checked or not. You have it so the checkbox is always not disabled even if it's checked.